src/plugins/crossref_event_data/schema.py

```diff
@@ -1,27 +1,56 @@
 from dateutil.parser import parse as parse_date_string
+import re
 
 from marshmallow import fields
 from marshmallow.exceptions import ValidationError
 
+from core.settings import Origins
+from core.logic import get_enum_by_name
+from processor.models import Uri
 from processor.schemas import EventSchema
 
 
+DOI_PATTERN = re.compile(r'10\.\d{4,9}/[-._;()/:A-Z0-9]+', re.I)
```

Comment:
where is this regex coming from? I remember that Crossref has the DOI regex somewhere in a blog post...

Reply:
Almost identical :sunglasses: 
https://www.crossref.org/blog/dois-and-matching-regular-expressions/

I must admit the regex here was there before I started on this project :sweat_smile: